### DISCUSSION TITLE
Alternate between two signals in a timed fashion?

### DISCUSSION THREAD

**Discussion by jschwalbe** [merbanan/rtl_433#2559] [Q&A]
Howdy!

I have two devices that I want to listen to readings from.
1) `rtl_433 -f 916.45M -s 1200k -R 223` - this device sends readings every 10 seconds, but only actually updates the readings once an hour.
2) `rtl_433 -f 915M` - this device sends readings every 12.5 minutes and I'd like to have an updated reading ~hourly.

My goal would be to have it listen for `1` for 30 seconds, enough to get at least one reading. Then it should listen for `2` for 13 minutes (or until it gets a reading), then sleep for half an hour and start over.

Is this possible? Thanks!

**Comment by AutomGuy** [comment]
Good Morning,
when you take a look into the help of rtl_433 you can find the following

`rtl_433 -f 916450000 -f 9150000 -H 120 -F syslog:127.0.0.1 1433`

-H means the time in seconds . -> The sdr stick reads 120 s the first frequency an the 120 s the second frequency and the from begin.

you have to save the transmission into a db or json files or some think like this and handed over to your GUI in a suitable way.
take a look into the documentation each sensor has a unique id. over this id you can identify the sensor.

**Comment by AutomGuy** [comment]
IMHO when you will use rtl_433 in your manner, you need to sdr sticks. 
here in rtl_433 project is described how to operate two SDR sticks on one computer.

**Comment by jschwalbe** [comment]
Have been unable to work on this for awhile, thanks for the suggestions @AutomGuy. 

I'm sending the data over mqtt, so the storage of info is not a concern. I just want to have it run one command for 15 seconds (to be sure it gets at least one reading from the every 10 second signal) and then run the other command for 13ish minutes (to be sure to get the every 12.5 min reading). And then repeat the process indefinitely.

**Reply by zuckschwerdt** [reply]
Then maybe loop three commands:
```
while : ; do
    rtl_433 -f 916.45M -s 1200k -R 223 -E quit
    rtl_433 -f 915M -E quit
    sleep 1800
done
```
